recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - pandas
     - sympy
     - scikit-learn
-    - openlibm <0.8.0
+    - openlibm
```

Comment:
I remember there being some segfaults from this upgrade, so need to test this part a bit more. Could this part be reverted, and the other change by itself be implemented?

Reply:
With out this at the moment, the build and tests will fail as in [594b7ac](https://github.com/conda-forge/pysr-feedstock/pull/52/commits/594b7acfede69ded5e783c0607ba094e089cb432).

Julia 1.8.0 uses openlibm v0.8.1:
https://github.com/JuliaLang/julia/blob/v1.8.0/deps/openlibm.version

Pinning openlibm at < 0.8.0 will thus cause issues with Julia 1.8.0.

Reply:
I unpinned the julia-feedstock a few days ago:
https://github.com/conda-forge/julia-feedstock/pull/230

Note that the pinning here is only for the "host" as part of the build process and not for "run".